res/mac/FreenetTray/src/FNAppDelegate.m

```diff
@@ -0,0 +1,59 @@
+/* 
+    Copyright (C) 2015 Stephen Oliver <steve@infincia.com>
+    
+    This code is distributed under the GNU General Public License, version 2.
+    
+    3rd party libraries may be distributed under an alternate Open Source license.
+    
+    See the LICENSE file included with this code for details.
+    
+*/
+
+
+#import "FNAppDelegate.h"
+
+#import "FNNodeController.h"
+#import "FNDropdownMenuController.h"
+
+#import "NSBundle+LoginItem.h"
+
+@interface FNAppDelegate ()
+    
+@end
+
+@implementation FNAppDelegate
+
+- (void)applicationDidFinishLaunching:(NSNotification *)aNotification {
+    
+	// load factory defaults for node location variables, sourced from defaults.plist
+	NSString *defaultsPlist = [[NSBundle mainBundle] pathForResource:@"defaults" ofType:@"plist"];
+	NSDictionary *defaultsPlistDict = [NSDictionary dictionaryWithContentsOfFile:defaultsPlist];
+
+	NSUserDefaults *defaults = [NSUserDefaults standardUserDefaults];    
+    [defaults registerDefaults:defaultsPlistDict];
+ 
+    
+    nodeController = [[FNNodeController alloc] init];
+
+	/* 
+        Check for first launch key, if it isn't there this is first launch and 
+        we need to setup autostart/loginitem
+    */
```

Comment:
That looks like a result of pasting part of it from another method in the old code where tabs were tabs, I use spaces 100% now. The IDE didn't display them differently so I missed the inconsistency, but I'll make sure they're consistent from now on.